ce/api/util.py

```diff
@@ -5,6 +5,7 @@
 import numpy.ma as ma
 from netCDF4 import Dataset
 from shapely.wkt import loads
+from modelmeta import *
```

Comment:
Normally this is what I do... and then you fix my code with named imports :wink: